docs/content.zh/release-notes/flink-1.13.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Release Notes - Flink 1.13"
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release Notes - Flink 1.13
+
+These release notes discuss important aspects, such as configuration, behavior, or dependencies,
+that changed between Flink 1.12 and Flink 1.13. Please read these notes carefully if you are
+planning to upgrade your Flink version to 1.13.
+
+### Failover
+
+#### Remove `state.backend.async` option.
+
+##### [FLINK-21935](https://issues.apache.org/jira/browse/FLINK-21935)
+
+The `state.backend.async` option is deprecated. Snapshots are always asynchronous now (as they were
+by default before) and there is no option to configure a synchronous snapshot any more.
+
+The constructors of `FsStateBackend` and `MemoryStateBackend` that take a flag for sync/async
+snapshots are kept for API compatibility, but the flags are ignored now.
+
+#### Unify Binary format for Keyed State savepoints
```

Comment:
Is there a pattern to which words are capitalized and which aren't? This doesn't seem to be title-case, and "Binary" doesn't sound like a proper noun to me either.